terraform/instances.tf

```diff
@@ -32,18 +32,18 @@ resource "aws_launch_configuration" "infra-demo-web-lc" {
     "${module.vpc.default_security_group_id}",
     "${aws_security_group.web.id}",
   ]
-
+  associate_public_ip_address = "${var.associate_public_ip_address}"
+  key_name = "${aws_key_pair.infra-demo-pub.key_name}"
   #iam_instance_profile = "${aws_iam_instance_profile.?????.name}"
 
-  key_name = "${aws_key_pair.infra-demo-pub.key_name}"
   lifecycle {
     create_before_destroy = true
   }
   root_block_device {
     volume_type           = "gp2"
     delete_on_termination = true
   }
-  enable_monitoring = true
+  enable_monitoring           = true
```

Comment:
There are a lot of spaces being added here.

It would be nice if having the terraform fmt checker acting as a linter.